bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2BridgeHandler.java

```diff
@@ -0,0 +1,595 @@
+     */
+    private void initializeAssets() {
+        logger.debug("initializeAssets() {} called", this);
+        synchronized (this) {
+            Clip2BridgeConfig config = getConfigAs(Clip2BridgeConfig.class);
+
+            String ipAddress = config.ipAddress;
+            if (Objects.isNull(ipAddress) || ipAddress.isEmpty()) {
+                logger.debug("initializeAssets() invalid ip address '{}'", config.ipAddress);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        "@text/offline.conf-error-no-ip-address");
+                return;
+            }
+
+            try {
+                Clip2Bridge.testSupportsClip2(ipAddress);
+            } catch (IllegalStateException e) {
+                logger.debug("initializeAssets() hub does not support clip 2");
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        "@text/offline.clip2.conf-error-clip2-not-supported");
+                return;
+            } catch (IllegalArgumentException e) {
+                logger.debug("initializeAssets() invalid ip address '{}'", ipAddress);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        "@text/offline.clip2.conf-error-invalid-ip-address");
+                return;
+            } catch (ApiException e) {
+                logger.debug("initializeAssets() communication error on '{}'", ipAddress);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                        "@text/offline.communication-error");
+                return;
+            }
+
+            HueTlsTrustManagerProvider trustManagerProvider = new HueTlsTrustManagerProvider(ipAddress + ":443",
+                    config.useSelfSignedCertificate);
+
+            if (trustManagerProvider.getPEMTrustManager() == null) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        "@text/offline.clip2.conf-error-certificate-load");
+                return;
+            }
+
+            trustManagerRegistration = FrameworkUtil.getBundle(getClass()).getBundleContext()
+                    .registerService(TlsTrustManagerProvider.class.getName(), trustManagerProvider, null);
+
+            String applicationKey = config.applicationKey;
+            applicationKey = Objects.nonNull(applicationKey) ? applicationKey : "";
+            clip2Bridge = new Clip2Bridge(httpClient, this, ipAddress, applicationKey);
+
+            assetsLoaded = true;
+        }
+
+        scheduler.submit(() -> checkConnection());
+    }
+
+    /**
+     * Called when the connection goes offline. Schedule a reconnection event.
+     */
+    public void onConnectionOffline() {
+        if (assetsLoaded) {
+            logger.debug("onOffline() ThingStatus:UNKNOWN");
+            updateStatus(ThingStatus.UNKNOWN);
```

Comment:
Shouldn't it be rather:
```suggestion
            updateStatus(ThingStatus.OFFLINE);
```

? UNKNOWN should only be used in `initialize()` when actual status is not yet known.